.github/workflows/cherry-pick.yml

```diff
@@ -0,0 +1,62 @@
+# This workflow validates commit messages in a pull request to ensure they follow a specific pattern.
+# The pattern is defined as follows:
+# - The commit message must contain the string "[cherry-pick/XXX]" where XXX is a valid label.
+# - The commit message must also contain "(cherry picked from commit <commit_hash>)" where <commit_hash> is a 40-character SHA-1 hash.
+# - Merge commits are ignored.
+#
+# Copyright (c) Microsoft Corporation.
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+
+name: Cherry-Pick Workflow
+
+on:
+  pull_request:
+    types:
+      - edited
+      - opened
+      - reopened
+      - synchronize
+  workflow_dispatch:
+
+jobs:
+  cherry-pick-job:
+    if: contains(github.event.pull_request.title, '[cherry pick]') || contains(github.event.pull_request.title, '[cherry-pick]')
```

Comment:
This is case insensitive?

Reply:
Yes

Reply:
https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/evaluate-expressions-in-workflows-and-actions#contains